README.md

```diff
@@ -1,136 +1,96 @@
+- A more controlled process, not only because of custom stages that will be run within the workflow (Fetch, Cluster, Align and Inference, for now), but also by providing all the available parameters for each stage in a single sight, listing the possibilities for each one.
+- A simpler way to get the results, as they will be stored in a ZIP that will be downloaded in the local machine after each requested process has finished. Even if it fails, a filtered log message will be displayed, so that the user has a clue of what went wrong.
 
-# **L**icense **T**erms
+## License terms
 
 This Django package is open source software made available under the Apache 2.0 License terms. Please see the LICENSE file for further details.
 
-# **I**nstallation **P**rocess
+## Installation
 
-- The first step is to clone this library. You can do it by typing this line in your console:
+The first step is to clone this library. You can do it by typing this line in your console:
 ```bash
 git clone https://github.com/nachorequeno/mevolib-ui.git
 ```
 
-- After that, you have to enter to the folder:
+Next, you have to install all of the dependencies:
 ```bash
-cd mevolib-ui
+pip install mevolib-ui/MEvoLibGUI/.
 ```
 
-- Then, you have to install all of the dependencies:
+After that, generate the secret key and save it in a file named `.env`. You can do it by running the following command:
 ```bash
-pip install -e.
+python manage.py shell -c "from django.core.management.utils import get_random_secret_key; print(f'SECRET_KEY = {get_random_secret_key()}')" > .env
 ```
 
-- When you are done with this, the next step make sure you have installed python-dotenv to store the secret key provided by Django.
-If you have not it, you can type:
+Now clone `mevolib-workflow` repository too:
 ```bash
-pip install python-dotenv
+git clone https://github.com/JAlvarezJarreta/mevolib-workflow.git mevolib-ui/MEvoLibGUI/nextflowFiles/mevolib-workflow
 ```
 
-- Before going into the next step, you have to go to the MEvoLibGUI folder:
+Finally, make sure you have redis-server installed. To do so, you can run
 ```bash
-cd MEvoLibGUI
+redis-server --version
 ```
-
-- The next step is to generate the secret key. You can do it by running:
-```bash
-python manage.py shell -c "from django.core.management.utils import get_random_secret_key; print(get_random_secret_key())"
+And it should return something like
 ```
-
-- Then, copy the value and create a file named ".env", like this:
-```bash
-touch .env
+Redis server v=7.2.4 sha=00000000:0 malloc=libc bits=64 build=2d86b7859915655e
 ```
+If that is not the case, please refer to the [Redis website](https://redis.io/docs/install/install-redis/) to install it.
 
-- Go to this folder, and paste this:
-```bash
-SECRET_KEY = (your secret key value)
-```
+You are now ready to use MEvoLib's interface!!
 
-- After that, clone the MEvoLib library in the MEvoLibGUI folder you were:
-```bash
-git clone https://github.com/JAlvarezJarreta/MEvoLib.git
```

Comment:
I talked to Nacho about this, and whether cloning the project is not needed, you should make sure the user installs MEvoLib library itself, so maybe adding a brief line refering to the repository and informing them to install it are the best we can do.

Reply:
MEvoLib is a dependency of this repository, so `pip` will deal with it already. I have added a link to the repo in the introduction following your suggestion.